samples/snippets/speech_to_storage_beta_draft.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START speech_transcribe_with_speech_to_storage_beta]
+
+from google.cloud import speech_v1p1beta1 as speech
+from google.cloud import storage
+import json
+
+
+def export_transcript_to_storage_beta(
+    input_storage_uri,
+    output_storage_uri,
+    encoding,
+    sample_rate_hertz,
+    language_code,
+    bucket_name,
+    bucket_prefix,
+):
+
+    # input_uri URI for audio file in Cloud Storage, e.g. gs://[BUCKET]/[FILE]
+    audio = speech.RecognitionAudio(uri=input_storage_uri)
+
+    # Pass in the URI of the Cloud Storage bucket to hold the transcription
+    output_config = speech.TranscriptOutputConfig(gcs_uri=output_storage_uri)
+
+    # Speech configuration object
+    config = speech.RecognitionConfig(
+        encoding=encoding,
+        sample_rate_hertz=sample_rate_hertz,
+        language_code=language_code,
+        audio_channel_count=2,
```

Comment:
How do we know it to be two channels, can we leave out the channel count from arguments.

Reply:
The audio file I used for testing is dual channel. 

If single channel specified:

```
google.api_core.exceptions.InvalidArgument: 400 Must use single channel (mono) audio, but WAV header indicates 2 channels.
```

I can use a single channel audio and remove the ```audio_channel_count``` argument.